### DISCUSSION TITLE
Hide upload file button  in asset field if Max Files  = 1

### DISCUSSION THREAD

**Discussion by lotarbo** [statamic/cms#8150] [General]
Currently, it permits uploading an unlimited amount)

https://github.com/statamic/cms/assets/1766890/8ab8704c-5763-4b43-818f-396327bb4433

Perhaps it's feasible to alter the behavior to match that of the Grid/Replicator field: concealing the Add Row button once the maximum count is reached ?

**Comment by jasonvarga** [comment]
You can upload multiple but it won't let you save it. You will get a validation error.

I'm pretty sure we used to prevent uploads and got feedback to keep them allowed. This gives you more flexibility in your workflow.
